lib/controller.php

```diff
@@ -77,9 +84,10 @@ public function pull($payload) {
 		}
 
 		// We add wpghs to commits we push out, so we shouldn't pull them in again
-		if ( 'wpghs' === substr( $payload->head_commit->message, -5 ) ) {
+		if ( 'wpghs' === substr( $payload->head_commit->message, - 5 ) ) {
```

Comment:
No, I've been using PHPStorm and it's done some of the formatting on its own, including, oddly, that one.